lib/facilities/ppms_client.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module Facilities
+  # Core class responsible for api interface operations
+  class PPMSClient < Common::Client::Base
+    configuration Facilities::PPMSConfiguration
+
+    def provider_locator(params)
+      qparams = build_params(params)
+      response = perform(:get, 'v1.0/ProviderLocator?', qparams)
+      bbox_num = params[:bbox].map { |x| Float(x) }
+      response.body.select! do |provider|
+        provider['Latitude'] > bbox_num[1] && provider['Latitude'] < bbox_num[3] &&
+          provider['Longitude'] > bbox_num[0] && provider['Longitude'] < bbox_num[2]
+      end
+      response.body
+    end
+
+    def provider_info(identifier)
+      qparams = { :$expand => 'ProviderSpecialties' }
```

Comment:
unfortunate param name -- might be more readable to use a string key here instead '$expand' but ultimately not a huge deal

Reply:
Also would be nice to have a comment linking to api documentation for the ppms api